src/main/java/org/jetbrains/research/anticopypaster/utils/KeywordsMetrics.java

```diff
@@ -0,0 +1,74 @@
+package org.jetbrains.research.anticopypaster.utils;
+
+import org.jetbrains.research.extractMethod.metrics.features.FeaturesVector;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class KeywordsMetrics extends Flag{
+
+    public KeywordsMetrics(List<FeaturesVector> featuresVectorList){
+        super(featuresVectorList);
+        calculateAverageKeywordsMetrics();
+    }
+
+    /**
+    This is a function that will get the keywords metric out of 
+    the FeaturesVector that is passed in
+    Keywords uses every odd-number value from metrics 17-78 
+     */
+    private float getKeywordsMetricFromFV(FeaturesVector fv){
+        if(fv != null){
+            float[] fvArray = fv.buildArray();
+            int totalKeywords = 0;
+            for(int i = 16; i<=77; i+=2){
```

Comment:
Nitpick but <=77 vs < 77 shouldn't make a difference so I'm not sure why that's there.